File: leetcode/java/1431_KidsWithTheGreatestNumberOfCandies.java
import java.util.*;

class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {

        List<Boolean> ans = new ArrayList<>();

        for (int i = 0; i < candies.length; i++) {
            boolean greatest = true;
            int temp = candies[i] + extraCandies;
            for(int j = 0; j < candies.length; j++) {
                if (candies[j] > temp) {
                    greatest = false;
                }
            }
            ans.add(greatest);
        }

        return ans;
    }
}

File: leetcode/go/1431_KidsWithTheGreatestNumberOfCandies.go
func kidsWithCandies(candies []int, extraCandies int) []bool {
	ans := make([]bool, 0, len(candies))
	for _, candy := range(candies) {
			givenExtra := candy + extraCandies
			temp := true
			for _, c := range (candies) {
					if givenExtra < c {
							temp = false
					}
			}
			ans = append(ans, temp)
	}
	return ans
}